pkg/sdk/common_types.go

```diff
@@ -237,13 +237,46 @@ const (
 	NullInputBehaviorStrict            NullInputBehavior = "STRICT"
 )
 
+// ToNullInputBehavior maps STRICT to RETURNS NULL ON NULL INPUT, because Snowflake returns RETURNS NULL ON NULL INPUT for any of these two options
+func ToNullInputBehavior(value string) (NullInputBehavior, error) {
+	switch strings.ToUpper(value) {
+	case string(NullInputBehaviorCalledOnNullInput):
+		return NullInputBehaviorCalledOnNullInput, nil
+	case string(NullInputBehaviorReturnsNullInput), string(NullInputBehaviorStrict):
+		return NullInputBehaviorReturnsNullInput, nil
+	default:
+		return "", fmt.Errorf("unknown null input behavior: %s", value)
+	}
+}
+
+var AllAllowedNullInputBehaviors = []NullInputBehavior{
+	NullInputBehaviorCalledOnNullInput,
```

Comment:
Left an explanation comment: `// ToNullInputBehavior maps STRICT to RETURNS NULL ON NULL INPUT, because Snowflake returns RETURNS NULL ON NULL INPUT for any of these two options
`

Reply:
Also, added a TODO: `// TODO [SNOW-1348103]: add a test documenting STRICT behavior`

Reply:
👍, I saw the comment, but just wondered if we are preventing STRICT option on purpose